styles.css

```diff
@@ -0,0 +1,516 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Lato', sans-serif;
```

Comment:
- [ ] I can see clearly that you used only Lato font throughout your website, but I will like to bring to your notice that COCOOGOOSE font was also used especially for intro section(hero texts) on the homepage. I will strongly encourage you to use COCOGOOSE fonts as well so that your UI fonts can match the design guideline.

Here is an example 

YOur design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/233505462-e634e11a-8fa8-4368-a47a-71f3ddb2b793.png)

design guideline 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/233505530-6896e05d-0bf6-4a6f-9569-2e7787ce7dfa.png)

cocogoose should have been used here and in other places as well. Please take your time to inspect the fonts 😄